WebDriverAgentLib/Categories/XCUIElement+FBTVFocuse.m

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2018-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "XCUIElement+FBTVFocuse.h"
+
+#import <XCTest/XCUIRemote.h>
+#import "FBApplication.h"
+#import "FBErrorBuilder.h"
+#import <FBTVNavigationTracker.h>
+#import "XCUIElement+FBUtilities.h"
+#import "XCUIElement+FBWebDriverAttributes.h"
+
+#if TARGET_OS_TV
+
+int const MAX_ITERATIONS_COUNT = 100;
+
+@implementation XCUIElement (FBTVFocuse)
+
+-(BOOL)fb_focuseWithError:(NSError**) error
+{
+  [[FBApplication fb_activeApplication] fb_waitUntilSnapshotIsStable];
+  if (self.wdEnabled) {
+    FBTVNavigationTracker *tracker = [FBTVNavigationTracker trackerWithTargetElement:self];
+    for (int i = 0; i < MAX_ITERATIONS_COUNT; i++) {
+      if (self.hasFocus) {
+        return YES;
+      }
+      if (self.exists) {
+        FBTVDirection direction = tracker.directionToMoveFocuse;
+        if(direction != FBTVDirectionNone) {
+          [[XCUIRemote sharedRemote] pressButton: (XCUIRemoteButton)direction];
+          continue;
+        }
+      }
+      [[[FBErrorBuilder builder] withDescription:@"Unable to reach element. Try to use XCUIRemote commands."]
+       buildError:error];
+      return NO;
```

Comment:
the previous function always returns NO, so you can just return its result instead of having the separate return operator